planner/core/task.go

```diff
@@ -46,6 +46,18 @@ var (
 	_ task = &mppTask{}
 )
 
+const (
+	// The size of a paging distsql grows from minPagingSize to maxPagingSize
+	minPagingSize  float64 = 64
+	maxPagingSize  float64 = 8192
+	pagingSizeGrow float64 = 2
+	// pagingGrowingSum is the sum of paging sizes during growing to the max page size
+	// pagingGrowingSum = (pagingSize ^ n - 1) * minPagingSize = (2 ^ 8 - 1) * 64 = 16320
```

Comment:
```suggestion
	// pagingGrowingSum = (pagingSizeGrow ^ n - 1) * minPagingSize = (2 ^ 8 - 1) * 64 = 16320
```

Reply:
`8` is a magic number here. One alternative is
```golang
const (
	// The size of a paging distsql grows from minPagingSize to maxPagingSize
	minPagingSize      int = 64
	maxPagingSizeShift int = 7
	maxPagingSize      int = minPagingSize << maxPagingSizeShift
	pagingSizeGrow     int = 2
	// pagingGrowingSum is the sum of paging sizes during growing to the max page size
	// pagingGrowingSum = (pagingSizeGrow ^ (maxPagingSizeShift + 1) - 1) * minPagingSize = (2 ^ 8 - 1) * 64 = 16320
	pagingGrowingSum int = ((2 << maxPagingSizeShift) - 1) * minPagingSize
```

Reply:
By using shifting, it supposes the growth factor is 2.